test/src/main/java/org/apache/accumulo/test/functional/AccumuloClientIT.java

```diff
@@ -40,12 +42,28 @@
 import org.apache.accumulo.core.singletons.SingletonManager;
 import org.apache.accumulo.core.singletons.SingletonManager.Mode;
 import org.apache.accumulo.harness.AccumuloClusterHarness;
+import org.junit.Before;
 import org.junit.Test;
 
 import com.google.common.collect.Iterables;
 
 public class AccumuloClientIT extends AccumuloClusterHarness {
 
+  @Before
+  public void deleteUsers() throws Exception {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      Set<String> users = client.securityOperations().listLocalUsers();
+      ClusterUser user1 = getUser(0);
+      ClusterUser user2 = getUser(1);
+      if (users.contains(user1.getPrincipal())) {
+        client.securityOperations().dropLocalUser(user1.getPrincipal());
+      }
+      if (users.contains(user2.getPrincipal())) {
+        client.securityOperations().dropLocalUser(user2.getPrincipal());
+      }
+    }
+  }
+
```

Comment:
I am curious as to why you added this method?  The current test passes for me on standalone without users.

Reply:
Nevermind, it was passing on Mini because i had the path wrong to the properties file.  I opened a ticket to make this better: https://github.com/apache/accumulo/issues/1168

Reply:
good question lets see if I can discern my reasoning from all the other IT tests I changed :).  
The test originally used a hardcoded user/password set and it was not deleted  so the next time it ran it failed.  I switched it to use the AccumuloClusterHarness.getUser(?) like many of the tests thinking it would be deleted but no, they still were not.  The harness expects them to be prefixed with the classname BUT the harness getUser() does not add the classname - go figure.  This is probably something that should be addressed in a separate ticket.  So I checked as to how it is being managed in other tests that use the getUser() and they explicity delete the user on startup.  I'm sure there are better ways to manage all this and will take any direction recommended.

Reply:
Ah OK thanks for the explanation.

Reply:
>  I switched it to use the AccumuloClusterHarness.getUser(?) like many of the tests thinking it would be deleted but no, they still were not.

@hkeebler I have been looking into this and I found the following code.  It seems some attempt to clean up user names with a certain prefix is made.  I am not sure what the intent was here.  Was the intent that the harness would clean up users?  Looking at `StandaloneAccumuloClusterConfiguration.java` (which to seems to create user names) it does not create user names with that prefix.  

https://github.com/apache/accumulo/blob/a1b1ae6bfef2ca11d92b86ab60930b3c56971f6e/test/src/main/java/org/apache/accumulo/harness/AccumuloClusterHarness.java#L227

I am not suggesting any course of action, just sharing what I have found.